Documentation/operations/upgrade.rst

```diff
@@ -478,6 +478,7 @@ Helm Options
   container images are not scheduled on non-Linux nodes.
 * ``cluster.id`` cannot be empty and a value must be specified.
   Use the ``0`` value to leave Cluster Mesh disabled.
+* ``bpf.hostLegacyRouting`` will be set to true automatically if ``cni.chainingMode`` is set to any other value than ``none`` (default) 
```

Comment:
Oh, I didn't notice before but this is in the wrong section since 1.12 is already released. Can you move this up into the "Helm Options" section for the upcoming 1.13 release notes?

Reply:
fixed